src/Client.php

```diff
@@ -21,6 +21,13 @@ public function deleteIndex($uid)
         return $this->indexInstance($uid)->delete();
     }
 
+    public function deleteAllIndexes()
```

Comment:
Since it's a method we provide now for our users, we should test it 🙂
- test when there is no index
- test when there are indexes

Reply:
closed with 67adb2f